src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs

```diff
@@ -16,373 +16,386 @@ public static class Sse
         /// <summary>
-        /// __int64 _mm_cvtss_si64 (__m128 a)
+        /// __int64 _mm_cvtss_si64 (__m128 a); CVTSS2SI r64, xmm/m32
         /// </summary>
         public static long ConvertToInt64(Vector128<float> value) => ConvertToInt64(value);
+
         /// <summary>
-        /// float _mm_cvtss_f32 (__m128 a)
+        /// float _mm_cvtss_f32 (__m128 a); HELPER: MOVSS
         /// </summary>
         public static float ConvertToSingle(Vector128<float> value) => ConvertToSingle(value);
 
         /// <summary>
-        /// __m128 _mm_cvtsi32_ss (__m128 a, int b)
+        /// __m128 _mm_cvtsi32_ss (__m128 a, int b); CVTSI2SS xmm, reg/m32
         /// </summary>
         public static Vector128<float> ConvertToVector128SingleScalar(Vector128<float> upper, int value) => ConvertToVector128SingleScalar(upper, value);
         /// <summary>
-        /// __m128 _mm_cvtsi64_ss (__m128 a, __int64 b)
+        /// __m128 _mm_cvtsi64_ss (__m128 a, __int64 b); CVTSI2SS xmm, reg/m64
         /// </summary>
         public static Vector128<float> ConvertToVector128SingleScalar(Vector128<float> upper, long value) => ConvertToVector128SingleScalar(upper, value);
 
         /// <summary>
-        /// int _mm_cvttss_si32 (__m128 a)
+        /// int _mm_cvttss_si32 (__m128 a); CVTTSS2SI r32, xmm/m32
         /// </summary>
         public static int ConvertToInt32WithTruncation(Vector128<float> value) => ConvertToInt32WithTruncation(value);
         /// <summary>
-        /// __int64 _mm_cvttss_si64 (__m128 a)
+        /// __int64 _mm_cvttss_si64 (__m128 a); CVTTSS2SI r64, xmm/m32
         /// </summary>
         public static long ConvertToInt64WithTruncation(Vector128<float> value) => ConvertToInt64WithTruncation(value);
 
         /// <summary>
-        /// __m128 _mm_div_ps (__m128 a,  __m128 b)
+        /// __m128 _mm_div_ps (__m128 a,  __m128 b); DIVPS xmm, xmm/m128
         /// </summary>
         public static Vector128<float> Divide(Vector128<float> left, Vector128<float> right) => Divide(left, right);
 
         /// <summary>
-        /// __m128 _mm_div_ss (__m128 a,  __m128 b)
+        /// __m128 _mm_div_ss (__m128 a,  __m128 b); DIVSS xmm, xmm/m32
         /// </summary>
         public static Vector128<float> DivideScalar(Vector128<float> left, Vector128<float> right) => DivideScalar(left, right);
 
         /// <summary>
-        /// __m128 _mm_loadu_ps (float const* mem_address)
+        /// __m128 _mm_loadu_ps (float const* mem_address); MOVUPS xmm, m128
         /// </summary>
         public static unsafe Vector128<float> LoadVector128(float* address) => LoadVector128(address);
 
         /// <summary>
-        /// __m128 _mm_load_ss (float const* mem_address)
+        /// __m128 _mm_load_ss (float const* mem_address); MOVSS xmm, m32
         /// </summary>
-        public static unsafe Vector128<float> LoadScalar(float* address) => LoadScalar(address);
```

Comment:
Fixed name to be `LoadScalarVector128`

Reply:
Not necessary, we do not have `LoadScalarVector256`, and `Scalar` already implies `Vector128`.

Reply:
I changed it here for consistency with the other APIs naming convention. I figured it was better to be consistent.

Reply:
Fair enough.

Reply:
I also prefer the addition of `Vector128` to the name; I think it makes it clearer, especially since when encountered in code it may not always be clear that it is returning a vector.